examples/quick_template_score_calc.ipynb

```diff
@@ -528,7 +520,7 @@
    "source": [
     "temperature_score = TemperatureScore(\n",
     "    time_frames = [ETimeFrames.LONG],\n",
-    "    scopes=[EScope.S1S2],\n",
+    "    scopes=[EScope.S1S2, EScope.S3],\n",
```

Comment:
@MichaelTiemannOSC i cannot find, where `CBRE` data comes from into the Jupiter notebook.
From cell 17, it seems like  `CBRE`  is supposed to come from 'data/20220927 ITR Tool Sample Data.xlsx', sheet "Portfolio".
But when i open this file locally - it has no row for `CBRE`.

Where does the data come from into Jupiter notebook?

Reply:
Must be some funny version skew.  My version of your sources from GitHub has CBRE data in the input data, target data, and portfolio pages.  It's at the very bottom--not in alphabetical order anymore.

Reply:
found it. This xls was open in Libreoffice on my machine, when i was switching between branches - maybe that's why the latest version didn't update as i expected.
all right now, i have data to work with

Reply:
@MichaelTiemannOSC figured out. It's because in Jupiter, we load benchmarks "EI_OECM_PC", not "EI_OECM_S3".
As a result, all S3 data is folded into S1S2, and weights are:
S1S2= 100%
S3 = 0%

In this PR, S3 is designed to work with S3 benchmarks.

Proposals?

Reply:
From slack chat with Michael:

The problem is that the benchmark data you are using in your tree has only S1S2 data in one case and only S3 data in another case.  The newer benchmark data I created define S1S2, S1S2S3, and S3 so you can do a true S1S2 + S3 analysis and also a full S1S2S3 combined analysis.  As I’ve explained before, the specific benchmarks describe what specific Scope3 data they really want to use, and the V2 template permits that.  For now, for your cases, it’s ok to pretend there’s just one scope 3 category called S3 and work with that.   It would be interesting to compare how a company’s S1S2 align with 1.5C (using only S1S2 budget) vs. S3 aligning with 1.5C (using only S3 budget) and how S1S2S3 aligns with S1S2S3 budget.

When you can show that, you will have the strong “S3-only” result envisioned.  After that, it makes sense to get my changes in (when they are ready!) and refine the work so that each sector us using/estimating the correct S3 categories from whatever company disclosures/sector averages are available.

Reply:
I see. Maybe I can make a new Jupiter notebook, to calculate S1S2 separately from S3, and display tables side by side. I will need to check, how I can do that

Reply:
I added a new jupiter notebook for that: `examples/s1s2_s3_calc.ipynb`